src/Sulu/Bundle/AdminBundle/DependencyInjection/SuluAdminExtension.php

```diff
@@ -144,16 +144,17 @@ public function load(array $configs, ContainerBuilder $container): void
         $configuration = $this->getConfiguration($configs, $container);
         $config = $this->processConfiguration($configuration, $configs);
 
-        $container->setParameter($this->getAlias() . '.name', $config['name']);
-        $container->setParameter($this->getAlias() . '.email', $config['email']);
-        $container->setParameter($this->getAlias() . '.user_data_service', $config['user_data_service']);
-        $container->setParameter($this->getAlias() . '.resources', $config['resources']);
-        $container->setParameter($this->getAlias() . '.collaboration_enabled', $config['collaboration']['enabled']);
-        $container->setParameter($this->getAlias() . '.collaboration_interval', $config['collaboration']['interval']);
-        $container->setParameter($this->getAlias() . '.collaboration_threshold', $config['collaboration']['threshold']);
-
-        $container->setParameter($this->getAlias() . '.forms.directories', $config['forms']['directories'] ?? []);
-        $container->setParameter($this->getAlias() . '.lists.directories', $config['lists']['directories'] ?? []);
+        $container->setParameter('sulu_admin.name', $config['name']);
+        $container->setParameter('sulu_admin.email', $config['email']);
+        $container->setParameter('sulu_admin.user_data_service', $config['user_data_service']);
+        $container->setParameter('sulu_admin.resources', $config['resources']);
+        $container->setParameter('sulu_admin.collaboration_enabled', $config['collaboration']['enabled']);
+        $container->setParameter('sulu_admin.collaboration_interval', $config['collaboration']['interval']);
+        $container->setParameter('sulu_admin.collaboration_threshold', $config['collaboration']['threshold']);
+
+        $container->setParameter('sulu_admin.forms.directories', $config['forms']['directories'] ?? []);
+        $container->setParameter('sulu_admin.lists.directories', $config['lists']['directories'] ?? []);
+        $container->setParameter('sulu_admin.templates.configuration', $config['templates']);
```

Comment:
here is some inconsistency with the parameter names

`$config['collaboration']['enabled']` translates to `sulu_admin.collaboration_enabled`
but 
`$config['forms']['directories']` translates to `sulu_admin.forms.directories`

Reply:
Really strange not sure why this is. I know that when 2.0 was build it was defined that service name should always be `<bundle>.<service>` and so no more as one dot should be inside them. Strange this wasn't done for the parameters here.